docs/API.md

```diff
@@ -62,6 +69,23 @@ components that are passed in or composed.
  * @return {Function} factory.compose Add stamp to stamp. Chainable.
  * @return {Function} factory.static Add properties to the factory object. Chainable.
 
+```js
+const stamp = stampit({
+  methods: {
+    aplify(value) {
+      return this.factor * value;
+    }
+  },
+  refs: {
+    defaultFactor: 1
+  },
+  init() {
+    this.factor = this.factor >= 0 ? this.factor : this.defaultFactor;
+  }
+});
+
+var objectInstance = stamp({factor: 1.1});
```

Comment:
I see a few variables being declared with `var`. Should do another pass and fix those. :)